modules/ROOT/pages/about-logging-of-payload-formats.adoc

```diff
@@ -112,14 +120,15 @@ You may want to force the exclusion of certain formats from the external logs. T
         globalPayloadExclusion: true
 ----
 
-You may also want to force the conversion and forwarding of formats that are considered to not be convertible.
+=== Include Payloads 
 
-[WARNING]
-Note that by doing this there's a risk of breaking the payload.
+You might want to convert and forward payload formats that Runtime Manager agent considers not convertible.
 
+[IMPORTANT]
+If you force conversion for all payload formats, the payload might break.
```

Comment:
What does it mean to have a payload "break"?

Reply:
Also...are the steps that follow doing this: forcing the conversion of all formats?